.gitignore

```diff
@@ -1 +1,2 @@
 docs/*.dia.autosave
+stem.sublime-workspace
```

Comment:
The sublime file is ignored here, but it's also added to the project. If the added file is a good starting point for sublime users, that's fine. Just commenting this to double check this is your intention @arendjr?